workflow/scripts/fasta_generate_genome.py

```diff
@@ -0,0 +1,10 @@
+nbr_chromosomes = snakemake.params[0]
+out_file = open(snakemake.output[0], "w")
+with open(snakemake.input[0], "r") as ref_idx:
+    for i in range(nbr_chromosomes):
+        line = ref_idx.readline()
+        fields = line.strip().split("\t")
+        chromosome = fields[0]
+        chromosome_len = fields[1]
+        print("{chr}\t{len}".format(chr=chromosome, len=chromosome_len), file=out_file)
```

Comment:
This is not a bed at all, isn't it?

Reply:
You are right. We just need the chromosomes and it lengths here.
So we can just cut the first two lines of the fasta index.